### ISSUE TITLE
Question about test iterations

### ISSUE THREAD

**Issue by generalmilk** [BVLC/caffe#799]
I recently modified ImageNet example to train on my own image data. However, I just realize I forgot to change some parameters. They are the test iteration and test batch size. My validation set is smaller. Adapting ImageNet's test iteration (1,000) and test batch size (50) will cover my validation set about 1.2 times in each test. My validation set is not shuffled, only training set does. My questions is, will this setting learn biased model? My validation set is 2,089 classes x 20 images per class, do I also have to set test batch size to 20?
Thank you!

**Comment by ashutosh33** [comment]
Hi,
I am new to Caffe. I would like to have some information about the following parameters from solver.prototxt file.
test_iter
test_interval
display
average_loss
snapshot.

It would be good if I have more information about this particular file.
